src/main/scala/org/globalforestwatch/summarystats/gladalerts/GladAlertsSummary.scala

```diff
@@ -10,34 +10,31 @@ import org.globalforestwatch.util.{Geodesy, Mercantile}
 import scala.annotation.tailrec
 
 /** LossData Summary by year */
-case class GladAlertsSummary(stats: Map[GladAlertsDataGroup, GladAlertsData] =
-                             Map.empty,
-                             kwargs: Map[String, Any])
+case class GladAlertsSummary(stats: Map[GladAlertsDataGroup, GladAlertsData] = Map.empty)
   extends Summary[GladAlertsSummary] {
 
   /** Combine two Maps and combine their LossData when a year is present in both */
   def merge(other: GladAlertsSummary): GladAlertsSummary = {
     // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
-    GladAlertsSummary(stats.combine(other.stats), kwargs)
+    GladAlertsSummary(stats.combine(other.stats))
   }
 }
 
 object GladAlertsSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
+  def getGridVisitor(kwargs: Map[String, Any]): GridVisitor[Raster[GladAlertsTile], GladAlertsSummary] = {
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)